### DISCUSSION TITLE
Has anyone successfully deployed firefly-iii on Azure Container Apps or ECS Fargate?

### DISCUSSION THREAD

**Discussion by jonathan-joestar420** [firefly-iii/firefly-iii#9193] [Firefly III]
I've recently become interested in serverless services like Azure Container Apps and ECS Fargate, as I believe they can help me reduce my cloud hosting costs.

Currently, I have a Firefly III instance running on Docker within an Azure VM. I'm trying to migrate this setup to Azure Container Apps. However, after pushing my Docker images to ACR and deploying them to Azure Container Apps, the application is no longer working (it was functioning correctly before the migration).

I'm not a professional developer, so I'm seeking assistance here. I'm wondering if anyone has experience with a similar migration? Could there be an issue with my steps?

Any insights or advice would be greatly appreciated!

**Comment by JC5** [comment]
I haven't done this before, but what are the errors you're getting?
